AmplifyPlugins/Auth/AWSCognitoAuthPlugin/AWSCognitoAuthPlugin+Configure.swift

```diff
@@ -80,7 +80,11 @@ extension AWSCognitoAuthPlugin {
     func identityPoolServiceConfiguration(from authConfiguration: JSONValue) -> AmplifyAWSServiceConfiguration? {
         let regionKeyPath = "CredentialsProvider.CognitoIdentity.Default.Region"
         guard case .string(let regionString) = authConfiguration.value(at: regionKeyPath) else {
-            Amplify.Logging.warn("Could not read Cognito identity pool information from the configuration.")
+            Amplify.Logging.info("""
+                Cognito identity pool information is missing from the configuration. This is expected if you are not
+                using Cognito identity pool, otherwise check your configuration to make sure you have the region
+                under `CognitoIdentity`.
```

Comment:
```suggestion
                Cognito Identity Pool information is missing from the configuration. This is expected if you are not
                using a Cognito Identity Pool. Otherwise, check your configuration to make sure you specify a
                `Region`, `PoolId`,  under `CognitoIdentity` > `Default`.
```

Not sure if this is actually clearer, but I'm trying to guide folks to make sure they include the correct values as noted in https://docs.amplify.aws/lib/auth/existing-resources/q/platform/ios. Maybe we just give them the doc link?

```suggestion
                Cognito Identity Pool information is missing from the configuration. This is expected if you are not
                using a Cognito Identity Pool. Otherwise, check your configuration to make sure you specify the
                required values. See https://docs.amplify.aws/lib/auth/existing-resources/q/platform/ios
```

Reply:
i think we can do both,
```
Cognito Identity Pool information is missing from the configuration. This is expected if you are not
using an Identity Pool, otherwise check your configuration to make sure you specify a `Region`,
`PoolId`, under `CognitoIdentity` > `Default`.
See https://docs.amplify.aws/lib/auth/existing-resources/q/platform/ios for more details.
```